index.html

```diff
@@ -63,6 +63,236 @@ <h1>
+            </p>
+            <div class="btn-container-flex">
+              <button type="button" class="btn-sm">html</button>
+              <button type="button" class="btn-md">css</button>
+              <button type="button" class="btn-lg">javaScript</button>
+            </div>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="vertical-space"></div>
+      </section>
+
+      <section class="about">
+        <h1>
+          About<br />Myself<br />
+          you!
+        </h1>
+        <p class="paragraph">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <span class="connect">LETS CONNECT</span>
+
+        <div class="socialmedia_icons_container">
+          <ul class="media_icons">
+            <li>
+              <a href="#">
+                <img src="images/icons/twitter.svg" alt="Twitter Icon" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/icons/linkedin.png" alt="LinkedIn Icon" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/icons/m.svg" alt="Media Icon" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/icons/cat.svg" alt="Github Icon" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/icons/fingers.svg" alt="Victory Icon" />
+              </a>
+            </li>
+          </ul>
+          <div class="btn-container about-btn">
+            <button type="button" class="btn-xxl btn-resume">
+              Get my resume
+            </button>
+          </div>
+        </div>
+      </section>
+
+      <section class="skill-sec">
+        <div class="skill-description">
+          <ul class="skill-flex">
+            <li>
+              <h3>Languages</h3>
+            </li>
+            <li>
+              <ion-icon name="chevron-down-sharp"></ion-icon>
+            </li>
+          </ul>
+        </div>
+        <div class="skill-container">
+          <div class="skill-1 gap">
+            <img
+              class="skill-img"
+              src="images/works/skill_1.png"
+              alt="javaScript"
+            />
+            <span class="text-skill">JAVASCRIPT</span>
+          </div>
+          <div class="skill-2">
+            <img
+              class="skill-img"
+              src="images/works/skill_2.png"
+              alt="javaScript"
+            />
+            <span class="text-skill">HTML</span>
+          </div>
+          <div class="skill-3">
```

Comment:
- [ ] I really appreciate your effort in putting the Technologies("Languages", "Frameworks", "skills") in list 👏. However, you should also put the technology belongings(for example: for languages we have Ruby, Javascript and Html) into a list. In order to do that, you will use the concept of **Nested lists**. As an example, look at piece of code below
```bash
<ul>
    <li>Languages
         <ul>
              <li>Javascript</li>
              <li>Ruby</li>
              <li>Html</li>
        </ul>
    </li>
</ul>
```
Note that this is just for illustration, do not copy and paste them. Just refer to it in order to understand what I wanted to mean and then use the technique to implement yours. You can also read this [documentation](https://www.webucator.com/article/how-to-properly-nest-lists-in-html/) to understand more about nested lists. 🏃

Reply:
@BertrandConxy thank you very much for your feedback. I am familiar with the concept of Nested Lists but in this instance, since the name **Languages** is supposed to be in the same line with the chevron icon I wanted to put the **Languages** and the **chevron right icon** in a single div so that I can apply flex properties to the div to align the elements in a straight line. Kindly explain how I can achieve this while applying the concept of Nested lists?

![question](https://user-images.githubusercontent.com/29442846/177079980-25ab121d-18a1-4608-bbdd-999608d30a33.png)